src/datadog_api_client/rest.py

```diff
@@ -237,6 +240,19 @@ def __init__(self, configuration):
         if configuration.proxy:
             proxy = aiosonic.Proxy(configuration.proxy, configuration.proxy_headers)
         self._client = aiosonic.HTTPClient(proxy=proxy)
+        self._configuration = configuration
+
+    def _retry(self, response, counter):
+        if (
+            not self._configuration.enable_retry
+            or counter >= self._configuration.max_retries
+            or response.status_code not in RETRY_AFTER_STATUS_CODES
```

Comment:
Nit: Would be nice to check the method here as well as we do with the REST client